recipes/diplib/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "diplib" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DIPlib/diplib/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 01c6fc164d973fe89a9a9b04082a2915aeaedfbd207bf78ce3f83aa298b603d7
+  patches:
+    - pydip.patch
+    - vendored_deps.patch
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Why no love for Windows? Is it the GUI stuff that's causing problems?

Reply:
I didn't need it myself, and since I don't have a local machine to debug, given the amount of devendoring needed and possible rough edges, I wasn't too excited about it 😬